### PR TITLE
Feature/add mustache views

### PR DESCRIPTION
**Pull request by sdonaghey03**

### CODE REVIEW

File: src/main/java/com/kainos/example/jaxws/services/IThingService.java [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package com.kainos.example.jaxws.services;
```
Comment (msjhall138) [inline src/main/java/com/kainos/example/jaxws/services/IThingService.java]: Extra blank space

File: src/main/java/com/kainos/example/jaxws/services/IThingService.java [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 
 
+import com.kainos.example.jaxws.models.Thing;
```
Comment (msjhall138) [inline src/main/java/com/kainos/example/jaxws/services/IThingService.java]: Unused import

File: src/main/java/com/kainos/example/jaxws/services/ThingService.java [hunk 3]
```diff
@@ -1,20 +1,22 @@
 package com.kainos.example.jaxws.services;
 
 import com.kainos.example.jaxws.models.Thing;
```
Comment (msjhall138) [inline src/main/java/com/kainos/example/jaxws/services/ThingService.java]: Unused import

**Review by msjhall138 — CHANGES_REQUESTED** [review]

File: src/main/java/com/kainos/example/jaxws/services/ThingService.java [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
 
     @Override
     public String getThing(int id) {
-        things.add(new Thing(thingCount, "This is thing: " + thingCount));
-        thingCount++;
-        return things.get(id).toString();
+        return thingModel.getThing(id).getDescription();
+    }
+
+    @Override
```
Comment (msjhall138) [inline src/main/java/com/kainos/example/jaxws/services/ThingService.java:17]: Ditto above

File: src/main/java/com/kainos/example/jaxws/services/ThingService.java [hunk 5]
```diff
@@ -1,20 +1,22 @@
...
-import java.util.ArrayList;
 
 @WebService(endpointInterface = "com.kainos.example.jaxws.services.IThingService")
 public class ThingService implements IThingService {
 
-    private int thingCount = 0;
-    private ArrayList<Thing> things = new ArrayList<Thing>();
+    private ThingModel thingModel = ThingModel.getInstance();
 
     @Override
```
Comment (msjhall138) [inline src/main/java/com/kainos/example/jaxws/services/ThingService.java:12]: Do not need override for interface implementation

**Review by msjhall138 — CHANGES_REQUESTED** [review]

File: src/main/java/com/kainos/example/jaxws/services/ThingService.java (hunk 5 above)
Comment (sdonaghey03) [reply src/main/java/com/kainos/example/jaxws/services/ThingService.java:12]: To be debated

Comment (msjhall138) [reply src/main/java/com/kainos/example/jaxws/services/ThingService.java:12]: Grand - leave it for now

**Comment by sdonaghey03** [comment]
Changes implemented.

**Review by msjhall138 — APPROVED** [review]

**Pull request merged**